Corona-Warn-App/src/main/res/values/styles.xml

```diff
@@ -606,7 +606,8 @@
     <style name="CertificateCardInfoButton" parent="buttonIcon">
         <item name="android:height">@dimen/button_icon</item>
         <item name="android:width">@dimen/button_icon</item>
-        <item name="android:padding">14dp</item>
+        <item name="android:padding">8dp</item>
```

Comment:
reducing the padding makes the clickable are smaller and harder to hit, especially for impaired users

Reply:
Yes, I remember we actually had a fix a while back that introduced a bigger padding exactly for this purpose.

Reply:
Do you have a ticket link, then I can close the current ticket with this reference

Reply:
You can change the position of the icon without changing its padding

Reply:
Fixed position

Reply:
negative margins seem hacky. You can remove the padding of the card, there are guidelines already.

Reply:
removed negative margin, but had also to remove the style because of having padding bottom, without further hacks